src/OrleansAzureUtils/Storage/AzureStorageUtils.cs

```diff
@@ -386,5 +389,38 @@ internal static string PrintStorageException(Exception exception)
                         (extendedError.AdditionalDetails != null && extendedError.AdditionalDetails.Count > 0) ?
                             String.Format(", ExtendedErrorInformation.AdditionalDetails = {0}", Utils.DictionaryToString(extendedError.AdditionalDetails)) : String.Empty);
         }
+
+        internal static JsonSerializerSettings ConfigureJsonSerializerSettings(IProviderConfiguration config)
```

Comment:
Also should probably provide a method without IProviderConfiguration config, so one can just use it as is (but still clone).